app/javascript/src/ActiveDocs/components/ActiveDocsSpec.jsx

```diff
@@ -1,18 +1,127 @@
 // @flow
 
 import React from 'react'
-
 import { createReactWrapper } from 'utilities/createReactWrapper'
 import SwaggerUI from 'swagger-ui-react'
-
 import 'swagger-ui-react/swagger-ui.css'
 
-type ActiveDocsSpecProps = {
+const AUTOCOMPLETE_PARAMS_MAP = {
+  'app_key': 'app_keys',
+  'app_id': 'app_ids',
+  'application_id': 'application_ids',
+  'user_key': 'user_keys',
+  'user_id': 'user_ids',
+  'account_id': 'account_ids',
+  'metric_name': 'metric_names',
+  'metric_id': 'metric_ids',
+  'backend_api_metric_name': 'backend_api_metric_names',
+  'service_id': 'service_ids',
+  'admin_id': 'admin_ids',
+  'service_plan_id': 'service_plan_ids',
+  'application_plan_id': 'application_plan_ids',
+  'account_plan_id': 'account_plan_ids',
+  'client_id': 'client_ids',
+  'client_secret': 'client_secrets',
+  'service_token': 'service_tokens',
+  'access_token': 'access_token'
+}
+
+type Primitive = string | number | boolean
+
+type AccountData = {
+  [string]: Array<{name: string, value: string}> | []
+}
+
+type SwaggerResponse = {
```

Comment:
Can't this type be imported from Swagger flow types?

Reply:
The problem is both
`flow-typed search swagger-ui`
And
`flow-typed search swagger-ui-react`
return
`No definitions found, sorry!`